src/components/ChatBeginingText.js

```diff
@@ -0,0 +1,59 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import styles from '../styles/styles';
+import Text from './Text';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+
+const propTypes = {
+
+    isDefaultChatRoom: PropTypes.bool,
+    chatUsers: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Please provide comments. I would suggest taking a look at StyleGuide and thus we can reduce the review period. It will help both you and the reviewers.